bigtable/beam/workload-generator/src/main/java/bigtable/WorkloadGenerator.java

```diff
@@ -88,19 +81,47 @@ private static void cancelJob(BigtableWorkloadOptions options, DataflowPipelineJ
     client.updateJob(jobId, job);
   }
 
-  public static class ReadFromTableFn extends AbstractCloudBigtableTableDoFn<Long, Void> {
+  public static class ReadFromTableFn extends DoFn<Long, Void> {
```

Comment:
Please add some doc explaining that this is needed to share the bigtable client between DoFn threads